docker-compose.yml

```diff
@@ -19,6 +19,14 @@ services:
     container_name: tno-database
     build:
       context: ${DB_CONTEXT:-db/postgres/docker}
+    deploy:
```

Comment:
Adding resource limits to pods to reduce overhead and debug why API keeps crashing after it grows in memory usage.